src/style.css

```diff
@@ -0,0 +1,47 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-child(1),
+.stars--2 > .stars__star:nth-child(1),
+.stars--3 > .stars__star:nth-child(1),
+.stars--4 > .stars__star:nth-child(1),
+.stars--5 > .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(2),
+.stars--3 > .stars__star:nth-child(2),
+.stars--4 > .stars__star:nth-child(2),
+.stars--5 > .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-child(3),
+.stars--4 > .stars__star:nth-child(3),
+.stars--5 > .stars__star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-child(4),
+.stars--5 > .stars__star:nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > .stars__star:nth-child(5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
why you need so many of them?
you need only five actually
2 point in checklist